eng/scripts/Language-Settings.ps1

```diff
@@ -411,7 +411,7 @@ function EnsureCustomSource($package) {
   return $package
 }
 
-function Update-dotnet-DocsMsPackages($DocsRepoLocation, $DocsMetadata) {
+function Update-dotnet-DocsMsPackages($DocsRepoLocation, $DocsMetadata, $PackageSourceOverride, $DocValidationImageId) {
```

Comment:
If these aren't used then we should not add the parameters.

Reply:
Closed this one for now.